app/models/manageiq/providers/ansible_tower/provider.rb

```diff
@@ -60,6 +62,18 @@ def url=(new_url)
     default_endpoint.url = uri.to_s
   end
 
+  def self.seed
+    embedded_in_zone ||
+      create_with(:url => "https://localhost:8443", :zone => Zone.default_zone)
```

Comment:
This will need a reliable way to determine how all the other appliances will be able to access this one.
Maybe something like [`MiqRegion#remote_ws_url`](https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_region.rb#L242-L245) will be required?